src/components/AddItems.js

```diff
@@ -72,7 +72,9 @@ const AddItemsToList = () => {
       db.add({
         token: userToken,
         items: [item],
-      });
+      })
+        .then(() => alert('successfully added'))
+        .catch((e) => console.log('error', e));
```

Comment:
Good job fixing this bug haha. Now it notifies even while adding the first item!

Reply:
yay!! 🎉

Reply:
😅 Yes, this is a nice fix!